extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -71,6 +68,10 @@ function TrackedCornerstoneViewport({
     };
   }, []);
 
+  useEffect(() => {
+    cineService.setCine({ id: viewportIndex });
```

Comment:
How does this work if I drop in a new series to this viewport? Will it continue to use the previous series settings?